Felix's April Serenity update (#930)
akeneo/pim-helpcenter: content/updates/2021-04/2-Mass-delete-records.md

---
pim_announcement_img: ./img/reference-entity-records-mass-delete.png
pim_announcement_alt_img: "Reference entity records mass delete screen"
pim_announcement_audience:
- EE
---

# Mass delete your reference entity records
::: meta-data type="New" features="Productivity" available="mid-April" in="EE" link-to-doc="../articles/enrich-your-reference-entity-records.html#delete-a-selection-of-records"

> Have you ever needed to clean up your reference entity records? If yes, you can now do this very quickly within just a few clicks directly from your reference entity record grid. Indeed, you can now delete several records from the same reference entity at the same time. No doubt this will save you a lot of time!

Reviewer: ```suggestion
Do you ever need to clean up your reference entity records? It's super easy now, you can do it very quickly in a couple of clicks directly from your reference entity record grid. Indeed, you can now delete several records from the same reference entity at the same time. No doubt this will save you a lot of time!
```

> Ever needed to do some clean up in the records of your reference entities? You can now do it very quickly with just a few clicks directly from within your reference entity record grid. Indeed, you can now delete several records from the same reference entity at once. No doubt it will save you a tremendous amount of time!

Reviewer 2: ```suggestion
Have you ever needed to clean up your reference entity records? If yes, you can now do this very quickly within just a few clicks directly from your reference entity record grid. Indeed, you can now delete several records from the same reference entity at the same time. No doubt this will save you a lot of time!```

Author: ```suggestion
Have you ever needed to clean up your reference entity records? If yes, you can now do this very quickly within just a few clicks directly from your reference entity record grid. Indeed, you can now delete several records from the same reference entity at the same time. No doubt this will save you a lot of time!
```
